AzureLib Utils v1.0.3

AzureDoom opened the pull request:
- Reapplys the Right click event with Bedrock animations fix from https://github.com/JannisX11/blockbench-plugins/pull/175 
- Adds about.md 
- Removes the formatter that reformatted the plugins.json
- Updates `nvmrc`
- Removes Azurelib field from pure bedrock animation files.
- Fixes Azurelib not forcing the new repo format.
- Adds members.yml 
- Bumps version to 1.0.3

~~I also noticed a bug with the plugin when reinstalling it from the browser, I did move it last update from `plugins` to `plugins/azurelib_utils`. Is the old 1.0.0 still cached in the `plugins` folder where the app checks for?~~

JannisX11 commented:
I added some details about how the new file structure works to this repository's readme file. I think you still need to adjust the min_version. Hope that helps!

AzureDoom commented:
> I added some details about how the new file structure works to this repository's readme file. I think you still need to adjust the min_version. Hope that helps!

Thanks will adjust!

AzureDoom commented:
> I added some details about how the new file structure works to this repository's readme file. I think you still need to adjust the min_version. Hope that helps!

Updated now!
